hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -283,16 +292,48 @@
       name: chip_usb_suspend
       desc: '''Verify that the USB device can detect the presence of VBUS from the USB host.
 
-            TBD.
+            - On the top of `chip_usb_fs_df_tx_rx`, add below at the end of the sequence.
+            - Configure USB device to enter `Suspend` state and ensure `link_suspend` interrupt is
+              triggered.
+            - Test these 2 power modes
+              - Only USB is in power-off, the other blocks are still powered.
+              - pwrmgr shuts down power for all the blocks that supports power-off.
+            - Resume the device and check the `link_resume` interrupt.
+            - Re-enable data transfer and ensure data correctness.
+            '''
+      milestone: V2
+      tests: []
+    }
+    {
+      name: chip_usb_sof
+      desc: '''Verify that USB can detect SOF and respond with `usb_ref_pulse_o` and
+            `usb_ref_val_o`.
+
+            - Configure to enable `usb_ref_disable`.
+            - Send a frame with the same frame number as the USB device to trigger `frame`
+              interrupt.
+            - Ensure `usb_ref_pulse_o` and `usb_ref_val_o` behave correctly.
+            - Stop sending any frame and check the `host_lost` interrupt. Ensure `use_ref_*` behave
+              correctly.
+            '''
+      milestone: V2
+      tests: []
+    }
+    {
+      name: chip_usb_wake_debug
+      desc: '''Verify that `usb_state_debug_i` can be read from the CSR
+
+            - Drive random value on `usb_state_debug_i`.
+            - Enable the CSR `wake_debug` returns correctly value.
```

Comment:
nit: Should this be "Check whether the CSR `wake_debug` returns the correct value."